### ISSUE TITLE
How are you calculating the 'data_stats' for vector_loss and angular_loss? 

### ISSUE THREAD

**Issue by err-or** [neelabhsinha/flame#2]
Hi

I noticed that in both your loss functions (vector and angular) you are sending an argument, `'data_stats'` which is coming from a pickle under the [metafiles](https://github.com/neelabhsinha/flame/tree/main/metadata/data_statistics). I think you are using the range of gaze values to normalize the gaze loss however, in the paper, the gaze loss equation does not have anything like that. Could you please explain why are you multiplying both the prediction and the target with the range and adding the minimum gaze value at the [end](https://github.com/neelabhsinha/flame/blob/f416f2993a557dadc3394bf08692f15f40c7fcaa/losses/vector_loss.py#L18)? and how did you calculate the maximum and minimum gaze values for each dataset?

**Comment by neelabhsinha** [comment]
Hi, you are correct. In the paper, all equations are implemented by us in normalized form. Therefore, at the end, I am multiplying and adding to denormalize it to absolute angles. 
Maximum and minimum gaze values can be calculated by getting the individual gaze angles across the training set and then finding its maximum and minumum.
